osu.Game.Tests/Visual/Online/TestSceneBeatmapSetOverlayDetails.cs

```diff
@@ -44,7 +44,7 @@ public void TestMetrics()
             AddStep("set second set", () => details.BeatmapSet = secondSet);
             AddAssert("ratings set", () => details.Ratings.Metrics == secondSet.Metrics);
 
-            BeatmapSetInfo createSet() => new BeatmapSetInfo
+            static BeatmapSetInfo createSet() => new BeatmapSetInfo
```

Comment:
This is a fatal bug in VS16.3/roslyn3.3 and fixed now. Compiler is now bound to SDK version, forcing SDK 3.1.100 is required for this fix.